client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/hooks/use-category-patterns-map.ts

```diff
@@ -40,7 +40,13 @@ const useCategoryPatternsMap = ( patterns: Pattern[] ) => {
 				const isPage = isPagePattern( pattern );
 
 				insert( allCategoryPatternsMap, category, pattern );
-				if ( isPage && pageCategoryPatternsMap.hasOwnProperty( category ) ) {
+				// @TODO: For the future feature to shuffle pages will have to remove isPriorityPattern( pattern ) from this condition.
+				// It's only being used here just to help testing priority pages as the rest are excluded.
+				if (
+					isPage &&
+					pageCategoryPatternsMap.hasOwnProperty( category ) &&
+					isPriorityPattern( pattern )
```

Comment:
If we remove this condition, the UI on Assembler v2 will show the first page in the category when there aren't any pages featured in that category.

I think it's better to only show featured pages on Assembler v2 to help with testing.